### DISCUSSION TITLE
Need help

### DISCUSSION THREAD

**Discussion by Happyhands55** [community/community#103064] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

Someone has all access to my phone and I have changed my passwords and Everytime I change my passwords they still have access to everything what can I do

**Comment by gil-son** [comment]
If you suspect that someone has unauthorized access to your phone despite changing passwords, here are some steps you can take to secure your device:

    Change Passwords Securely:
        Use a strong, unique password that includes a mix of uppercase and lowercase letters, numbers, and symbols.
        Avoid using easily guessable information such as birthdays, names, or common words.
        Change passwords for all accounts associated with your phone.

    Enable Two-Factor Authentication (2FA):
        Activate two-factor authentication on your accounts whenever possible. This adds an extra layer of security by requiring a verification code sent to your mobile device or another trusted method.

    Check and Remove Suspicious Apps:
        Review the list of installed apps on your phone and uninstall any unfamiliar or suspicious ones.
        Be cautious of apps that request excessive permissions or have poor reviews.

    Update Your Operating System:
        Ensure that your phone's operating system and all installed apps are up-to-date. Manufacturers often release updates that include security patches.

    Scan for Malware:
        Use reputable antivirus or anti-malware software to scan your phone for any malicious software. Remove any threats detected.

    Review Account Activity:
        Check the recent activity on your accounts for any unusual login attempts. Many online services provide activity logs that can help you identify suspicious behavior.

    Secure Your Wi-Fi Network:
        Ensure that your home Wi-Fi network is password-protected and uses strong encryption (WPA3 is recommended).

    Consider Factory Reset:
        If the issue persists, consider performing a factory reset on your phone. This will erase all data, so make sure to back up important information first.

    Contact Customer Support:
        If you are still experiencing issues, contact the customer support of the services and platforms you are using. They may be able to assist you in securing your accounts.

    Report to Authorities:
        If you believe your accounts are being accessed unlawfully, consider reporting the incident to the appropriate law enforcement agency.

**Comment by ghostinhershell** [comment]
Hi @Happyhands55, 👋🏾 We really appreciate you flagging this. The best route to get this to the proper GitHub team is to use our [abuse reporting tools](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam). Here's all the info:

- [Reporting abuse or spam](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam)
- [Reporting a user](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-a-user)
- [Reporting an issue or pull request](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-an-issue-or-pull-request)
- [Reporting a comment](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-a-comment)
- You can also open a ticket on our [Support page](https://support.github.com/contact).

You can report behavior and content that violates community guidelines and terms. We are going to close this post, but for this and any future incidents, please refer to the links above.

Thank you!

**Comment by ghostinhershell** [comment]
Hey there!  👋

Thanks for posting in the GitHub Community, @Happyhands55! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category. The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them.

I've gone ahead and moved this to the correct category for you. Good luck!
